ord_schema/visualization/filters.py

```diff
@@ -159,12 +200,10 @@ def _temperature_conditions_html(temperature):
     txt = ''
     if (temperature.control.type == temperature.control.UNSPECIFIED
             or temperature.control.type == temperature.control.AMBIENT):
-        return 'ambient temperature<br>'
+        return 'ambient temperature'
     setpoint = units.format_message(temperature.setpoint)
     if setpoint:
         txt += f'{setpoint}'
-    if txt:
-        txt += '<br>'
     return txt
 
 
```

Comment:
In _analysis_format() below, we're still returning strings with angle brackets. Will those need escaping for HTML injection? Or is Jinja taking care of that?

Reply:
Jinja [does not auto-escape by default](https://jinja.palletsprojects.com/en/2.11.x/templates/#html-escaping), but it can. The need to handle HTML injections is an excellent point. If possible, it would be better for these filters to never return HTML tags so we could

Reply:
Flask escapes by default; I removed HTML tags from the bits of the template that were touched by my example, but clearly I missed some!

Reply:
Fixed (in another PR).